packages/visualizations/addon/templates/components/number-format-dropdown.hbs

```diff
@@ -0,0 +1,11 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for
+terms. --}}
+{{#basic-dropdown horizontalPosition='right' onClose=(action 'updateColumnNumberFormat') as |dd| }}
```

Comment:
Is there a reason why we need to update the format in the column object `onClose`? Can we do this `onUpdateFormat`?

Reply:
This is caused by the rendering issue we discussed before. If we update the column format immediately, the dropdown will be disappeared.

Reply:
ok